app/forms/validation/utils/ConstraintUtil.scala

```diff
@@ -16,21 +16,21 @@
 
 package forms.validation.utils
 
-import play.api.data.validation.{Constraint, Valid, ValidationResult}
+import play.api.data.validation.{Constraint, ValidationResult}
 
 
 object ConstraintUtil {
 
   def constraint[A](f: A => ValidationResult): Constraint[A] = Constraint[A](name = "")(f)
 
-//  implicit class ConstraintUtil[A](cons: Constraint[A]) {
-//
-//    def andThen(newCons: Constraint[A]): Constraint[A] =
-//      constraint((data: A) =>
-//        cons.apply(data) match {
-//          case Valid => newCons.apply(data)
-//          case r => r
-//        }
-//      )
-//  }
+  //  implicit class ConstraintUtil[A](cons: Constraint[A]) {
```

Comment:
do we need this?

Reply:
We don't for now, that is why I commented it out. I will delete it and we can add it when is needed.

Reply:
Actually I will delete it with the work I do now as I don't want to commit it against a scala update